Dfe.Academies.External.Web/Program.cs

```diff
@@ -1,17 +1,24 @@
+using Dfe.Academies.External.Web.Routing;
 using Microsoft.AspNetCore.Authentication.Cookies;
 using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
 
 var builder = WebApplication.CreateBuilder(args);
 ConfigurationManager configuration = builder.Configuration;
```

Comment:
It would, I'm not a big fan of var myself unless you're newing up an object so it's really clear what type it is. Here are the Microsoft style guidelines on it:

https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#implicitly-typed-local-variables